js-arcade/tictactoe/TicTacToe.js

```diff
@@ -16,7 +19,55 @@ function TicTacToe() {
             this.xIsNext = !this.xIsNext;
         }
     };
-    this.isGameOver = function(x, y) {
+    this.isGameOver = function (x, y) {
+        let current_player = this.xIsNext ? true : false;
+        //Check row
+        for (i = 0; i < width; i++) {
```

Comment:
I could be wrong, but I think you want to use let or var, otherwise i will be a global variable.